documentation/index.markdown

```diff
@@ -45,6 +45,8 @@ Puppet Server is the next-generation application for managing Puppet agents.
     * [Static file content](./puppet-api/v3/static_file_content.markdown)
 * **Status API endpoints**
     * [Status services](./status-api/v1/services.markdown)
+* **Metrics API endpoints**
+    * [Metrics API versions](./metrics_api.markdown)
```

Comment:
Maybe could just call this `Metrics service` instead of `Metrics API versions`, for consistency with the "status" service reference above?